spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/pubsub/GcpPubSubAutoConfiguration.java

```diff
@@ -381,4 +383,19 @@ public SubscriptionAdminClient subscriptionAdminClient(
 	public TransportChannelProvider transportChannelProvider() {
 		return InstantiatingGrpcChannelProvider.newBuilder().build();
 	}
+
+	/**
+	 * Configuration for reactive support.
+	 */
+	@Configuration
+	@ConditionalOnClass(Flux.class)
+	protected static class ReactivePubSubAutoconfiguration {
+
+		@Bean
+		@ConditionalOnMissingBean
```

Comment:
Should there be a property preventing this object from instantiating?

Reply:
For what reason? 
Doesn't look like this light object is some kind of overhead for the target application.

Reply:
Right, it's not necessary to keep it off by default because it's just a wrapper around a `SubscriberTemplate`. 
But I am also going to add a default scheduler bean into autoconfig and extract reactive configuration into a separate file, at which point it will make sense to add an overridable `matchIfMissing=true` property `spring.cloud.gcp.pubsub.reactive.enabled`.